projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-external-media.php

```diff
@@ -467,6 +482,17 @@ public function update_attachment_meta( $id, $item ) {
 				'post_excerpt' => $item['caption'],
 			)
 		);
+
+		if ( ! empty( $item['meta'] ) ) {
+			// Only allow meta properties defined in the schema.
+			// None of the validation or sanitization functions are doing this for me.
+			$meta_allowed  = array_keys( $this->media_schema['properties']['meta']['properties'] );
+			$meta_filtered = array_intersect_key( $item['meta'], array_flip( $meta_allowed ) );
```

Comment:
It's too bad `additionalProperties` doesn't cover that. I wonder if it doesn't work recursively but only in a top-level object?

Reply:
I believe that's the case.